_posts/2018-09-23-sr2019-announced.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: news
+title: SR2019 Competition Announced
+---
+
+We are pleased to announce the details of the 2019 Student Robotics competition!
```

Comment:
I'm always worried that we might sound a bit too formal in announcement emails, maybe 
`s/we are/we're/g`? just to lighten the tone.

Reply:
c9d1824